src/common/tooltip-utils/use-tooltip.ts

```diff
@@ -26,6 +27,10 @@ export const useTooltip = ({ onExpand, onCollapse, initialExpanded = false }: Us
         if (onCollapse) {
             onCollapse()
         }
+
+        if (hostRef?.current) {
+            hostRef.current.focus()
+        }
```

Comment:
```suggestion
        hostRef?.current?.focus()
```

Reply:
Thanks, updated. eslint used to fail in the past, considering it as a ternary, it seems it got fixed 👍